README.md

```diff
@@ -2,42 +2,1104 @@
 
 Observer adapter for [blindata.io](https://blindata.io/).
 
-Blindata is a SaaS platform that leverages Data Governance and Compliance to empower your Data Management projects.
-The purpose of this adapter is to keep the business glossary within Blindata constantly updated. Upon the occurrence of a creation, deletion, or modification of a dataproduct, Blindata is immediately and automatically notified to ensure that its catalog remains aligned.
+Blindata is a SaaS platform that leverages Data Governance and Compliance to empower your Data Management projects. The
+purpose of this adapter is to keep the data catalog within Blindata constantly updated. Upon the occurrence of a
+creation, deletion, or modification of a dataproduct, Blindata is immediately and automatically notified to ensure that
+its catalog remains aligned.
 
-*_This project have dependencies from the project [odm-platform](https://github.com/opendatamesh-initiative/odm-platform)_
+*_This project has dependencies from the
+project [odm-platform](https://github.com/opendatamesh-initiative/odm-platform)_
 
-# Run it
+      address: The address of ODM Notification Service
+
+## Contents
+
+1. [General Schema Annotations](#general-schema-annotations)
+    - [Entity](#entity)
+    - [Fields](#fields)
+2. [Metadata Mapping in Blindata](#metadata-mapping-in-blindata)
+    - [Systems](#systems)
+    - [DatastoreAPI](#datastoreapi)
+        - [JSON Schema](#json-schema)
+            - [Physical Entity](#physical-entity)
+            - [Physical Field](#physical-field)
+    - [AsyncAPI](#asyncapi)
+        - [Avro](#avro)
+            - [Physical Entity](#from-avro-to-physical-entity)
+            - [Physical Field](#from-avro-to-physical-field)
+3. [Examples](#examples)
+    - [DatastoreAPI](#datastore-api-example)
+        - [Single Entity](#single-entity)
+        - [Multiple Entities](#multiple-entities)
+     - [AsyncAPI](#async-api)
+        - [Raw Port Async Api V3](#raw-port-async-api-v3)
+        - [Raw Port Async Api V2](#raw-port-async-api-v2)
+        - [Entities Async Api V3](#entities-async-api-v3)
+        - [Entities Async Api V2](#entities-async-api-v2)
+4. [Prerequisites](#prerequisites)
```

Comment:
Tutta la parte su come eseguire il progetto la metterei sotto un'unico cappello, tipo "Run the project"
Sposterei anche le due sezioni con le configurazioni prima di "Test it", sotto il titolo "Configure the project" o qualcosa di simile